scripts/apps/search/components/ItemList.jsx

```diff
@@ -29,6 +31,10 @@ export class ItemList extends React.Component {
         this.setSelectedComponent = this.setSelectedComponent.bind(this);
         this.modifiedUserName = this.modifiedUserName.bind(this);
         this.setNarrowView = this.setNarrowView.bind(this);
+
+        keyboardManager.bind('x', () => {
```

Comment:
That's a good suggestion to make the code simple. I tried it, but did not work as the 'this' object is used inside multiSelectCurrentItem() function and it goes 'undefined' at the time of invocation. There could be a name for this concept (which could probably make my explanation better), but, I do not know what it is :).